internal/admin/static/player.css

```
@@ -1085,3 +1085,64 @@ body.has-player-bar main { padding-bottom: var(--player-bottom-inset); }
+  width: 20px; height: 20px; display: block;
+  fill: none; stroke: currentcolor;
+  stroke-width: 1.6; stroke-linecap: round; stroke-linejoin: round;
+}
+
+.source-status {
+  display: inline-flex; align-items: center; gap: 5px;
+  font-size: 0.75rem; color: var(--fg-muted);
+}
+.source-dot {
+  width: 7px; height: 7px; border-radius: 50%;
+  background: currentColor; flex: 0 0 auto;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)